parcels/particle.py

```diff
@@ -117,6 +117,7 @@ class ScipyParticle(_Particle):
     lon = Variable('lon', dtype=np.float32)
     lat = Variable('lat', dtype=np.float32)
     time = Variable('time', dtype=np.float64)
+    trajectory = Variable('trajectory', dtype=np.int32)
```

Comment:
I wonder whether we want to call this variable `id` instead of `trajectory`, since it will potentially be used for more than indexed trajectory output in the future?